src/compressed_tensors/utils/match.py

```diff
@@ -37,8 +38,8 @@
 
 def match_named_modules(
     model: torch.nn.Module,
-    targets: Iterable[str],
-    ignore: Iterable[str] = tuple(),
+    targets: Optional[Iterable[str]] = None,
```

Comment:
```suggestion
    targets: Optional[Iterable[str]]
```
This function doesn't really make any sense if the user doesn't have targets to match with